ui/page/staking/stake_overview.go

```diff
@@ -194,21 +205,22 @@ func (pg *Page) Layout(gtx C) D {
 
 func (pg *Page) layoutDesktop(gtx C) D {
 	pg.scroll.OnScrollChangeListener(pg.ParentWindow())
-
-	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-		layout.Rigid(pg.stakePriceSection),
-		layout.Flexed(1, func(gtx C) D {
-			return layout.Inset{Top: values.MarginPadding8}.Layout(gtx, func(gtx C) D {
+	return pg.Theme.List(pg.scrollContainer).Layout(gtx, 1, func(gtx C, i int) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(pg.stakePriceSection),
+			layout.Rigid(pg.stakeStatisticsSection),
+			layout.Rigid(func(gtx C) D {
 				if pg.showMaterialLoader {
 					gtx.Constraints.Min.X = gtx.Constraints.Max.X
 					return layout.Center.Layout(gtx, pg.materialLoader.Layout)
 				}
 				return pg.scroll.List().Layout(gtx, 1, func(gtx C, i int) D {
+					gtx.Constraints.Max.Y = ticketHeight
```

Comment:
Someone who has tickets should verify if this is necessary. What happens if it's removed?